### PR TITLE
Devops Recommendations Sec 1 to 5

### PR DESCRIPTION
**Pull request by myp192**
#1106  I took off some content to simply the recs. @gcharest @smellems Tried to fix the glitches but Travis is still failing.

### CODE REVIEW

**Review by gcharest — COMMENTED** [review]
Nice, thanks for the updated package!

So, if we are now quoting the DORA site, I guess we need to make that clear like:

From tDORA Technical Section (open quotes) text and elements (close quotes).

I'll try to do a first section and we can go from there?

File: _pages/en/capabilities.md [hunk 1]
```diff
@@ -167,9 +247,14 @@ Average of "How much time" questions
 - I can dynamically increase or decrease the cloud resources available for the service or product that I primarily support based on demand. [1-7]
 - I can monitor or control the quantity and/or cost of cloud resources used by the service or product that I primarily support. [1-7]
 
-#### Recommendations  
+#### Recommendations
+
+To achieve more rapid, reliable releases, and higher levels of availability, velocity, and reliability of the **Cloud Infrastructure**, the key actions need to be executed to imp
```
Comment (smellems) [inline _pages/en/capabilities.md:252]: I think this sentence is incomplete

Comment (myp192) [reply _pages/en/capabilities.md]: Thanks. Just made the small edit

**Review by smellems — APPROVED** [review]

**Review by gcharest — COMMENTED** [review]
Just wondering if the recommendations shouldn't be the ones in the DevOps Guides? There seems to be somewhat of an alignment but rewritten in our own words?

**Comment by gcharest** [comment]
Just to summarize the changes:
- Updated intro comments to recommendations to indicate when it was verbatim from the guides (`This excerpt from...`) vs when it was written in our own words (`Based on the DORA guides...`)
- Updated content that was verbatim to show it was a quote (add blockquote)
- Fixed cspell
- Fixed lint (spacing/indents, etc.)

**Pull request merged**